.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Custom issue
+about: Use this template just if your issue don't fit as a bug report or a issue template
```

Comment:
Please change: 
- "don't fit" to "doesn't fit" 
- "a issue template" to "an issue template"

Reply:
ow, "issue template" was a little mistake. the correct is a feature report

Reply:
done